### ISSUE TITLE
[FEATURE REQUEST]

### ISSUE THREAD

**Issue by hades200082** [valheimPlus/ValheimPlus#549]
**Is your feature request related to a problem? If so, please describe.**
When I have the mods _Valheim Plus_ and _ServerSide Simulations_ both installed, none of the changes from Valheim Plus take effect. No item pulling/depositing, no GUI changes, nothing.

**Describe the solution you'd like**
I'd like to be able to use ServerSide Simulations to have the game server run simulations rather than offloading them to the first client in an area. This will help me because the group I play with is split between UK, East-coast US and west-coast US, thus having an areas simulations be "powered" by one players PC causes excessive lag for players further from that player. Hosting the server centrally in New York provides everyone with acceptable ping to the server, but this doesn't make a difference unless the simulations happen on the server.

I'd like to also be able to use V+ for the QoL tweaks it offers.

**Comment by hades200082** [comment]
Just closed without comment? Does that mean it's done, not done, won't do?

**Comment by nxPublic** [comment]
It means that we are already in contact with the developers behind Serverside Simulations since multiple months.

We have offered to help them out to every extend they need or to tell us what to change to be more compatible with their work.

I personally haven't heard a update about this in quite the while, so I assumed this has been resolved by now.

**Comment by ddormer** [comment]
This has been resolved in the latest update to Serverside Simulations.

The problem in question is that [this code](https://github.com/valheimPlus/ValheimPlus/blob/development/ValheimPlus/GameClasses/Smelter.cs#L151)  tries to access `m_localPlayer` which isn't available on the server.

Ideally the `m_localPlayer` check is removed from the Smelter patch in ValheimPlus directly, but we don't have the time to ensure that it won't break anything else and so we've patched the ValheimPlus code from within Serverside Simulations and haven't had any complaints from current users thus far.
